### PR TITLE
Add schnaidt_2017_a_4147

### PR DESCRIPTION
**Pull request by DunklesArchipel**
The CV of a Pt(111) electrode was recorded in 0.5 M H2SO4. The electrode was prepared under UHV conditions.

I intend to use this on the main page since Pt(111) is a well-known system in the community. 

The figure should be included in #155  and the link should point to this entry.

### CODE REVIEW

File: literature/schnaidt_2017_a_4147/schnaidt_2017_a_4141.bib [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@article{schnaidt_2017_a_4141,
+  title={A combined UHV-STM-flow cell set-up for electrochemical/electrocatalytic studies of structurally well-defined UHV prepared model electrodes},
+  author={Schnaidt, Johannes and Beckord, Stephan and Engstfeld, Albert K and Klein, Jens and Brimaud, Sylvain and Behm, Rolf J{\"u}rgen},
+  journal={Physical Chemistry Chemical Physics},
+  volume={19},
+  number={6},
+  pages={4166--4178},
+  year={2017},
+  publisher={Royal Society of Chemistry}
+}
```
Comment (DunklesArchipel) [inline literature/schnaidt_2017_a_4147/schnaidt_2017_a_4141.bib:10]:
```suggestion
}

```

File: literature/schnaidt_2017_a_4147/schnaidt_2017_a_4141_f2_solid.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  version: 1
+  process:
+  - role: curator
+    name: Albert Engstfeld
+    orcid: https://orcid.org/0000-0002-9686-3948
+    date: 2022-05-23
+  - role: reviewer
+    name: Albert Engstfeld
+    orcid: https://orcid.org/0000-0002-9686-3948
+    date: 2022-01-21
```
Comment (DunklesArchipel) [inline literature/schnaidt_2017_a_4147/schnaidt_2017_a_4141_f2_solid.yaml:80]: could the reviewer please update this section

Comment (DunklesArchipel) [inline literature/schnaidt_2017_a_4147/schnaidt_2017_a_4141_f2_solid.yaml:80]:
```suggestion
```

File: literature/schnaidt_2017_a_4147/schnaidt_2017_a_4141_f2_solid.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  electrodes:
+    configuration: three
+    reference electrode:
+      source:
+        supplier: homemade
+      type: RHE
+    working electrode:
+      crystallographic orientation: '111'
+      material: Pt
+      preparation procedure: The elctrode was prepared under UHV conditions by sputtering the electrode surface and subsequent controlled heating and colling cycles, including oxygen adsorption during the colling process. The final step only involved controlled heating and cooling.
```
Comment (DunklesArchipel) [inline literature/schnaidt_2017_a_4147/schnaidt_2017_a_4141_f2_solid.yaml:22]:
```suggestion
      preparation procedure: The elctrode was prepared under UHV conditions by sputtering the electrode surface and subsequent controlled heating and cooling cycles, including oxygen adsorption during the cooling process. The final step only involved controlled heating and cooling.
```

**Review by saraedum — APPROVED** [review]

**Pull request merged**